File: bookManageBackDev/src/main/java/com/Httb/bookManage/controller/FirstItemSystemController.java
package com.Httb.bookManage.controller;

import com.Httb.bookManage.mbg.entity.Firstitemsystem;
import com.Httb.bookManage.service.FirstItemSystemService;
import com.Httb.bookManage.util.RequestPageData;
import com.Httb.bookManage.util.ResponsePageData;
import org.springframework.web.bind.annotation.*;
import org.springframework.web.multipart.MultipartFile;

import javax.annotation.Resource;

@RestController
public class FirstItemSystemController {
    @Resource
    private FirstItemSystemService firstItemSystemService;

    @PostMapping("/selectSystemList")
    public ResponsePageData<Firstitemsystem> selectSystemList(@RequestBody RequestPageData<Firstitemsystem> requestPageData) {
        return firstItemSystemService.selectSystemList(requestPageData);
    }

    @PostMapping("/insertSystem")
    public int insertSystem(@RequestBody Firstitemsystem firstitemsystem) {
        return firstItemSystemService.insert(firstitemsystem);
    }

    @PostMapping("/insertSystemOfExcel")
    public String insertSystemOfExcel(@RequestBody MultipartFile file) throws Exception {
        return firstItemSystemService.insertSystemOfExcel(file);
    }
}

File: bookManageFrontDev/src/services/FirstItemSystemController.js
import request from '@/utils/request' 

/*
*@param: requestPageData:com.Httb.bookManage.util.RequestPageData<com.Httb.bookManage.mbg.entity.Firstitemsystem>
*/
export function selectSystemList(requestPageData) {
    return request({
        url: request.baseUrl + `/selectSystemList`,
        method: 'POST',
        data:requestPageData
    })
}
/*
*@param: firstitemsystem:class com.Httb.bookManage.mbg.entity.Firstitemsystem
*/
export function insertSystem(firstitemsystem) {
    return request({
        url: request.baseUrl + `/insertSystem`,
        method: 'POST',
        data:firstitemsystem
    })
}
/*
*@param: file:interface org.springframework.web.multipart.MultipartFile
*/
export function insertSystemOfExcel(file) {
    return request({
        url: request.baseUrl + `/insertSystemOfExcel`,
        method: 'POST',
        data:file
    })
}

